test_crates/struct_field_marked_deprecated/new/src/lib.rs

```diff
@@ -0,0 +1,50 @@
+/// Normal case, should trigger the lint
+pub struct Plain {
+    #[deprecated]
+    pub field_becomes_deprecated: i64,
+    #[deprecated = "This field will be removed in the next major version"]
+    pub field_with_deprecation_message: String,
+}
+
+pub struct Tuple(
+    #[deprecated] pub i64,
+    #[deprecated = "Use new_tuple_field instead"] pub String,
```

Comment:
Could you make sure to run `cargo fmt` on the test crates to standardize their formatting and make them easier to read? I think this would get reformatted to put the attributes on a separate line.

Reply:
I think I ran the formatter, let me try it again

Reply:
Still the same bahavior

Reply:
Ah my bad, this is a tuple struct not a plain struct, and their formatting rules are different.